scripts/train/train.py

```diff
@@ -528,16 +532,16 @@ def main(cfg: DictConfig) -> Trainer:
     print('Building eval loader...')
     eval_icl_seq_len: int = icl_seq_len if icl_seq_len else max_seq_len
     evaluators, _, eval_gauntlet_callback = build_evaluators(
-        eval_loader_config,
-        icl_tasks_config,
-        eval_gauntlet_config,
+        eval_loader_config,  # TODO: async eval should not even call eval loader
```

Comment:
whats the todo here? don't quite understand

Reply:
in async eval we don't pass the icl / gauntlet configs into `build_evaluators`. There's a little bit more work needed to integrated into https://github.com/mosaicml/llm-foundry/pull/742 

Imagining we wouldn't even call build_evaluators in this case

Reply:
ohhh got it, makes sense. Yeah, please update the comment, thanks!